shared/actions/tracker.js

```diff
@@ -377,6 +386,10 @@ function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: bool
     start: ({username, sessionID, reason}) => {
       sessionIDToUsername[sessionID] = username
 
+      if (reason && (reason.reason === profileReason)) {
```

Comment:
we inject a custom reason if we're popping this up so we don't get trackers by accident